contribs/gnodev/cmd/gnodev/main.go

```diff
@@ -311,7 +311,7 @@ func execDev(cfg *devCfg, args []string, io commands.IO) (err error) {
 
 var helper string = `
 A           Accounts - Display known accounts and balances
-H           Help - Display this message
+H           Help - Also https://github.com/gnolang/gno/blob/master/docs/gno-tooling/cli/gnodev.md
 R           Reload - Reload all packages to take change into account.
 Ctrl+R      Reset - Reset application state.
 Ctrl+C      Exit - Exit the application
```

Comment:
@grepsuzette  I like the idea !

Some suggestions:
- Keep the short flag consistent in describing what each keystroke does. Consider moving this information above the short flags.
- Maintain the text within an 80-column length as it will fit in most terminal windows.
- Use direct documentation instead of referring to the README.

```go
var helper string = `For more in-depth documentation, visit the GNO Tooling CLI documentation:
https://docs.gno.land/gno-tooling/cli/gno-tooling-gnodev.

A           Accounts - Display known accounts and balances
H           Help - Display this message
R           Reload - Reload all packages to take change into account.
Ctrl+R      Reset - Reset application state.
Ctrl+C      Exit - Exit the application
`
```

### Suggestion preview 
![Screenshot 2024-07-04 at 08 55 22](https://github.com/gnolang/gno/assets/8671905/4e06f5e8-f390-476d-aaa4-e834e01249bc)

Reply:
Ok then, let's go for your version!

> For more in-depth documentation, visit the GNO Tooling CLI documentation: https://docs.gno.land/gno-tooling/cli/gno-tooling-gnodev. 

@leohhhn perhaps we could consider this kind of url for cli tools doc: https://docs.gno.land/tools/cli/gnodev  ?

Reply:
@grepsuzette there is an open issue about this: https://github.com/gnolang/gno/issues/2262

Reply:
Oh, perfect. Thank you!